File: api/DataStore/ChatacterVoiceSetting/CevioAIVoiceSetting/Talker2V40/Talker2V40.py


from pydantic import BaseModel, Field


class Talker2V40(BaseModel):
    Cast: str = Field(default="")
    Volume: int = Field(ge=0, le=100, default=50)
    Speed: int = Field(ge=0, le=100, default=50)
    Tone: int = Field(ge=0, le=100, default=50)
    Alpha: int = Field(ge=0, le=100, default=50)
    ToneScale: int = Field(ge=0, le=100, default=50)

File: app-ts/src/ZodObject/DataStore/ChatacterVoiceSetting/CevioAIVoiceSetting/Talker2V40/Talker2V40.ts
import { z } from "zod";
export const Talker2V40 = z.object({
  Cast: z.string().default(""),
  Volume: z.number().int().gte(0).lte(100).default(50),
  Speed: z.number().int().gte(0).lte(100).default(50),
  Tone: z.number().int().gte(0).lte(100).default(50),
  Alpha: z.number().int().gte(0).lte(100).default(50),
  ToneScale: z.number().int().gte(0).lte(100).default(50),
});
export type Talker2V40 = z.infer<typeof Talker2V40>;
